### PR TITLE
fix unhandled exception on scan timeout

### PR DESCRIPTION
**Pull request by abmantis**

### CODE REVIEW

File: custom_components/zha_map/neighbour.py [hunk 1]
```diff
@@ -119,13 +119,17 @@ def _update_info(self):
...
+                status, val = await self.device.zdo.request(
+                    zdo_t.ZDOCmd.Mgmt_Lqi_req, idx, tries=3, delay=1
+                )
+                self.debug("neighbor request Status: %s. Response: %r", status, val)
+                if zdo_t.Status.SUCCESS != status:
+                    self.supported = False
+                    self.debug("device does not support 'Mgmt_Lqi_req'")
+                    return
+            except asyncio.TimeoutError:
+                self.debug("'Mgmt_Lqi_req' timedout")
```
Comment (Adminiuga) [inline custom_components/zha_map/neighbour.py:132]:
```suggestion
                self.debug("[%s]: 'Mgmt_Lqi_req' timedout", self.device.nwk)
```

Comment (abmantis) [reply custom_components/zha_map/neighbour.py:132]: Shouldn't the nwk log be added to the log function instead? It is already printing the IEEE.

Comment (Adminiuga) [reply custom_components/zha_map/neighbour.py:132]: yep, you are right. I thought we were not logging any id at all. This looks good.

**Comment by Adminiuga** [comment]
Reverting this. It breaks "failed devices" logic. Time out exceptions are supposed to be handled in https://github.com/zha-ng/zha-map/blob/9ef57df257cfebaadc3d297fa4de2f664c9a5184/custom_components/zha_map/__init__.py#L138

**Pull request merged**